crates/ide/src/inlay_hints/closing_brace.rs

```diff
@@ -156,6 +156,47 @@ pub(super) fn hints(
     None
 }
 
+fn format_function_hint(func: &ast::Fn, max_length: usize) -> Option<String> {
```

Comment:
Imo we should do this as described in my comment in https://github.com/rust-lang/rust-analyzer/pull/19207#issuecomment-2850163723 instead